src/widget.rs

```diff
@@ -4,10 +4,22 @@ use ratatui::{
     style::{Color, Modifier, Style},
     widgets::{Block, Clear, Widget},
 };
-use vt100::Screen;
 
 use crate::state;
 
+pub trait Screen {
```

Comment:
Since the traits are intended to be used by consumers, please do add documentation comments.